.github/workflows/rebase-suite-e.yml

```diff
@@ -0,0 +1,45 @@
+name: Rebase the suite-e branch on develop
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  rebase:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Fetch the suite-e branch
```

Comment:
this is not really needed, as you can do it with 
```
      - name: Checkout code
        uses: actions/checkout@v3
        with:
          ref: "suite-e"
```
this will checkout the suite-e branch and then you can do rebase with `git rebase origin/develop` without needing to fetch the branch locally.

Reply:
f43c2e4ac4